brave/browser-laptop
# Torrent viewer: Add 'Stop Download' button

## Test Plan

1. Click a torrent or magnet link from this list https://webtorrent.io/free-torrents
2. Click 'Start Download'. The torrent detail view should appear.
3. Wait a few seconds, click 'Stop Download'. The torrent start page with a 'Start Download' button should return, indicating the download has stopped.


## Description
### NOTE: Depends on https://github.com/brave/browser-laptop/pull/7351 being merged first

Fixes: https://github.com/brave/browser-laptop/issues/6768

![stop-download](https://cloud.githubusercontent.com/assets/121766/24274254/4ee81842-0fe4-11e7-9ed1-067132ea551d.gif)
